### DISCUSSION TITLE
Immich hosted service

### DISCUSSION THREAD

**Discussion by awoimbee** [immich-app/immich#3651] [General]
Hi,
I love immich, I love open source, but I don't want to spend time hosting it, especially with backups and 100% uptime.
To me it seems obvious for the immich team to create a hosted service, but maybe it's too much of a bother, or maybe it's too early (in which case, is there a timeline) ?

TL;DR:
- is it planned ?
- if yes, when; if no, why ?

**Comment by oddlama** [comment]
I don't speak for the immich team, but just considering what this would cost - both in amount of money and required work hours for setup and maintenance - I would say that this is not going to happen anytime soon. This is probably more than a full time job to maintain, and most devs here want to work on immich instead of hosting such a service. It would truly require a small company to keep up.

Also, if the service were to be monetized, all (?) of the nice ML features cannot be provided to users because of the model licenses.

So even though it might exist sometime in the future, I wouldn't expect anything in that regard before immich is officially considered stable and there are no more desired features to be added - which may be never :P 
Ultimately, it's Alex's choice of course. At this time a reasonable expectation is that this is highly unlikely to exist anytime soon.

**Comment by alextran1502** [comment]
You can use service such as [Pikapods](https://www.pikapods.com/) to run Immich on 😀

**Reply by awoimbee** [reply]
I didn't know about pikapods, thanks !
